de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/activities/SendBundleActivity.java

```diff
@@ -0,0 +1,28 @@
+package de.fu_berlin.inf.dpp.activities;
+
+import com.thoughtworks.xstream.annotations.XStreamAlias;
+import com.thoughtworks.xstream.annotations.XStreamAsAttribute;
+import de.fu_berlin.inf.dpp.session.User;
+import java.io.File;
+import org.apache.commons.lang3.NotImplementedException;
+
+@XStreamAlias("SendBundleActivity")
+public class SendBundleActivity extends AbstractActivity {
+  @XStreamAsAttribute File bundleFile;
```

Comment:
I do not know how XStream marshall this but I guess it will not marshall the file content itself.